CHANGELOG.md

```diff
@@ -10,6 +10,60 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+Work in this release was contributed by @tjhiggins, @GrizliK1988, @davidturissini, @nwalters512, @aloisklink, @arturovt, @benjick, @maximepvrt, @mstrokin, and @kunal-511. Thank you for your contributions!
```

Comment:
should we move this into the `9.0.0-alpha.0` section? Or wait till the full release?

Reply:
I would wait for the full release!